database/seed.js

```diff
@@ -61,14 +61,22 @@ var save = () => {
         }
       };
 
-      Review.findOneAndUpdate(filter, update, {new: true, upsert: true}, function(err, doc) {
-        if (err) {
-          console.log(err);
-        } else {
-          console.log('Data saved!');
-        }
-      })
+    Review.findOneAndUpdate(filter, update, {new: true, upsert: true}, function(err, doc) {
+      if (err) {
+        console.log(err);
+      } else {
+        console.log('Data saved!');
+      }
+    });
   };
 };
 
-save();
\ No newline at end of file
+save();
+
+var getAll = (callback) => {
+  Review.find().exec(callback);
+};
+
+module.exports = {
+  getAll: getAll
+};
```

Comment:
Try object destructuring- you can do:


module.exports = {getAll} 

This will export an object with a key getAll and a value of your function :)

Reply:
Thanks!